druid-derive/Cargo.toml

```
@@ -17,3 +17,4 @@ proc-macro2 = "1.0.9"
 
 [dev-dependencies]
 druid = { path = "../druid", version = "0.6.0" }
+float-cmp = "0.6.0"
```

Can we avoid the `num-traits` dependency?
```suggestion
float-cmp = { version = "0.6.0", default-features = false }
```

---

Sure, we don't need it for this PR.